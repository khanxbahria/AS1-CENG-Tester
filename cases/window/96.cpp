#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
Tab K = Tab("aaa", "xxx","p");
wind->moveActiveTabTo(1);
Tab E = Tab("yyy", "z","eee");
wind->print();
Tab Z = Tab("x", "ddd","hh");
wind->newTab(E);
Tab J = Tab("r", "mmm","oo");
wind->newTab(J);
Tab H = Tab("d", "k","uu");
wind->changeActiveTabTo(1);
Tab S = Tab("oo", "www","oo");
wind->moveActiveTabTo(2);
Tab P = Tab("u", "www","nn");
wind->newTab(H);
Tab B = Tab("jj", "tt","gg");
wind->getActiveTab();
wind->closeTab();
Tab T = Tab("ggg", "uuu","h");
wind->newTab(B);
Tab F = Tab("xx", "k","r");
wind->closeTab();
wind->newTab(F);
Tab L = Tab("o", "w","rr");
wind->changeActiveTabTo(0);
wind->getActiveTab();
Tab O = Tab("uuu", "ee","ww");
wind->getActiveTab();
Tab D = Tab("ddd", "aaa","nnn");
wind->closeTab();
wind->moveActiveTabTo(4);
wind->newTab(D);
Tab N = Tab("i", "ee","hh");
wind->newTab(K);
Tab A = Tab("ss", "mmm","k");
wind->isEmpty();
Tab M = Tab("x", "mmm","a");
wind->changeActiveTabTo(1);
wind->moveActiveTabTo(4);
wind->changeActiveTabTo(2);
Tab W = Tab("jj", "kkk","hhh");
wind->changeActiveTabTo(0);
wind->newTab(A);
wind->isEmpty();
Tab X = Tab("cc", "zz","l");
wind->newTab(L);
wind->moveActiveTabTo(0);
Tab U = Tab("v", "v","cc");
wind->changeActiveTabTo(1);
Tab Q = Tab("nn", "x","kk");
wind->closeTab();
Tab C = Tab("a", "ppp","ttt");
wind->moveActiveTabTo(2);
wind->changeActiveTabTo(4);
Tab Y = Tab("kk", "lll","yyy");
wind->changeActiveTabTo(0);
Tab V = Tab("jj", "bbb","aa");
wind->changeActiveTabTo(0);
wind->print();
wind->moveActiveTabTo(3);
wind->moveActiveTabTo(2);
wind->newTab(C);
wind->moveActiveTabTo(0);
wind->isEmpty();
wind->newTab(M);
Tab R = Tab("qqq", "n","g");
wind->moveActiveTabTo(3);
wind->isEmpty();
wind->print();
wind->getActiveTab();
wind->moveActiveTabTo(3);
Tab G = Tab("ll", "g","ss");
wind->newTab(G);
wind->print();
wind->print();
wind->print();
wind->changeActiveTabTo(3);
wind->newTab(N);
wind->changeActiveTabTo(3);
wind->isEmpty();
wind->newTab(O);
wind->changeActiveTabTo(4);
wind->changeActiveTabTo(4);
wind->getActiveTab();
wind->newTab(P);
wind->closeTab();
wind->moveActiveTabTo(1);
wind->moveActiveTabTo(0);
wind->changeActiveTabTo(5);
wind->getActiveTab();
wind->changeActiveTabTo(1);
wind->getActiveTab();
wind->moveActiveTabTo(5);
wind->getActiveTab();
wind->closeTab();
wind->moveActiveTabTo(1);
wind->getActiveTab();
wind->getActiveTab();
wind->changeActiveTabTo(5);
wind->getActiveTab();
wind->moveActiveTabTo(5);
wind->isEmpty();
wind->closeTab();
wind->changeActiveTabTo(3);
wind->closeTab();
wind->moveActiveTabTo(5);
wind->newTab(Q);
wind->isEmpty();
wind->closeTab();
wind->changeActiveTabTo(2);
wind->getActiveTab();
wind->moveActiveTabTo(4);
wind->moveActiveTabTo(1);
wind->moveActiveTabTo(2);
wind->moveActiveTabTo(2);
Tab I = Tab("v", "b","bbb");
wind->moveActiveTabTo(3);
wind->isEmpty();
wind->print();
wind->isEmpty();
wind->closeTab();
wind->newTab(I);
wind->closeTab();
wind->newTab(R);
wind->isEmpty();
wind->closeTab();
delete wind;
}
	return 0;
}