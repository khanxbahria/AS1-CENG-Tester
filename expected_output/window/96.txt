The active tab is tab 0




The active tab is tab 0
w
o
rr

mmm
r
oo

mmm
ss
k

aaa
ddd
nnn

xxx
aaa
p

k
xx
r

The active tab is tab 3
ppp
a
ttt

xxx
aaa
p

k
xx
r

mmm
x
a

mmm
ss
k

mmm
r
oo

aaa
ddd
nnn

w
o
rr

The active tab is tab 4
ppp
a
ttt

xxx
aaa
p

k
xx
r

mmm
x
a

g
ll
ss

mmm
ss
k

mmm
r
oo

aaa
ddd
nnn

w
o
rr

The active tab is tab 4
ppp
a
ttt

xxx
aaa
p

k
xx
r

mmm
x
a

g
ll
ss

mmm
ss
k

mmm
r
oo

aaa
ddd
nnn

w
o
rr

The active tab is tab 4
ppp
a
ttt

xxx
aaa
p

k
xx
r

mmm
x
a

g
ll
ss

mmm
ss
k

mmm
r
oo

aaa
ddd
nnn

w
o
rr

The active tab is tab 3
ee
i
hh

g
ll
ss

mmm
ss
k

xxx
aaa
p

mmm
r
oo

mmm
x
a

aaa
ddd
nnn

w
o
rr

